taichi/backends/metal/api.cpp

```diff
@@ -133,6 +134,8 @@ size_t get_max_total_threads_per_threadgroup(
 
 bool is_metal_api_available() {
 #ifdef TI_PLATFORM_OSX
+  if (!get_environ_config("TI_WITH_METAL", true))
```

Comment:
I feel like `TI_ENABLE_X` sounds better: the library is still compiled **with** the backends, just sometimes we want to **disable** part of them.